lib/wickedbase.pm

```diff
@@ -148,6 +150,23 @@ sub get_current_ip {
     return;
 }
 
+=head2 download_all
+
+Download all files from data/wicked into WICKED_DATA_DIR.
+This method is used by before_test.pm.
+=cut
+sub download_all {
+    my $dst  = WICKED_DATA_DIR . '/data_wicked.tar.gz';
+    my $path = hashed_string($dst);
+    my $cmd  = "tar -czf $path -C '" . $bmwqemu::vars{CASEDIR} . "/data' wicked";
+
+    system($cmd) == 0 or die("Failed to create archive for wicked data ($cmd)");
+    assert_script_run("mkdir -p `dirname $dst`");
```

Comment:
wouldn't be easier just
mkdir -p WICKED_DATA_DIR ?

Reply:
yes :)